pytradfri/device.py

```diff
@@ -221,7 +220,7 @@ def set_predefined_color(self, colorname, *, index=0):
             color = COLORS[colorname.lower().replace(" ", "_")]
             return self.set_hex_color(color, index=index)
         except KeyError:
-            _LOGGER.debug('Could not match color name')
+            raise PyTradFriError("Could not match color name in COLOR_NAMES")
```

Comment:
Either create a new exception that inherits from this general error (ie `InvalidColor` error) or use the built-in exception for invalid values: `ValueError`.